### DISCUSSION TITLE
Activation of copilot failed on vscode

### DISCUSSION THREAD

**Discussion by Palkers76** [community/community#38950] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have installed copilot extention on vscode and tried to authenticate free copilot on my private account and.
getting error:
GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."
NOTE: I have other github account connected as repo, ( working for client ), not sure if I can still pay on my private account for copilot and use it with code from multiple repos.

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by Palkers76** [reply]
Thanks, 
I have to first signoff from my client github account, 
signin to my private account where I have copilot enabled,
signin again to client github.

this way it works , but pity that copilot extention is buggy and not allowing to authenticate to different account, same way as github pull request and issues extension.

**Reply by Palkers76** [reply]
wondering if it will work if I would setup ssh authentication on both github , or copilot works only on http ?

**Reply by hros** [reply]
I have a similar issue
I have a couple of github accounts and I can't connect to copilot on a new computer (copilot is still working on my other computer)
when I install the copilot extension, I see two popup messages:
1. `GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."`
2. `No access to GitHub Copilot found.` with a button labeled `Signup for GitHub Copilot`
When I press the button in the 2nd message, I see my copilot page (in my main account) but nothing seems to be activated on vs-code

when clicking on the output log (on the 1st message), I see:
```
[INFO] [auth] Invalid copilot token: missing token: 403 
[ERROR] [default] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."
[INFO] [auth] Invalid copilot token: missing token: 403 
[ERROR] [default] GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found."
```
any idea what I need to do to connect copilot on vs code to a specific github account?

**Comment by jasperdunn** [comment]
Still getting this issue, tried the steps above but it still displays the 403 message after logging in again.

**Reply by renko80** [reply]
The same for me, 

![image](https://user-images.githubusercontent.com/120383335/207047338-7b50ebb3-c916-4416-a100-0b4bbe9d6207.png)


I wonder if this is so tricky to log-in, how it would be to use it.....

**Comment by DrDinhLuong** [comment]
Hi all,

Me too with this  issue.

**Comment by rm-asif-amin** [comment]
@jasperdunn Did you manage to solve it?

**Reply by jasperdunn** [reply]
From memory it just magically started working again 🤷🏻‍♂️

Maybe try reinstalling vscode from scratch

**Reply by rm-asif-amin** [reply]
Thanks for the response. I had to activate free trial a second time from Github settings.
